src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  return getAvarage(
+    people
+      .filter(person => person.sex === 'm')
+      .filter(man => !century || getCentury(man) === century)
+      .map(({ died, born }) => died - born)
+  );
+}
+
+function getCentury(person) {
+  return Math.ceil(person.died / 100);
+}
+
+function getAvarage(numbers) {
+  return numbers.reduce((a, b) => a + b, 0) / numbers.length;
```

Comment:
Явно вказуй, що в цих змінних зберігається
```suggestion
  return numbers.reduce((aссum, age) => accum + age, 0) / numbers.length;
```